src/components/CIPipelineN/MountFromHost.tsx

```diff
@@ -0,0 +1,119 @@
+                (mountPathMap, index) => {
+                    return (
+                        <>
+                            <div className="mount-row mb-4 mt-4">
+                                <div className="fw-6 fs-13 lh-32 cn-7 "></div>
+                                <input
+                                    className="bcn-1 en-2 bw-1 pl-10 pr-10 pt-6 pb-6"
+                                    autoComplete="off"
+                                    placeholder="File path on Host"
+                                    type="text"
+                                    onChange={(e) => handleMountPath(e, index)}
+                                    value={mountPathMap[MountPathMap.FILEPATHONDISK]}
+                                    name={MountPathMap.FILEPATHONDISK}
+                                />
+                                <div className="flex bw-1 en-2">:</div>
+                                <input
+                                    className="bcn-1 en-2 bw-1 pl-10 pr-10 pt-6 pb-6"
+                                    autoComplete="off"
+                                    placeholder="File path on container"
+                                    type="text"
+                                    onChange={(e) => handleMountPath(e, index)}
+                                    value={mountPathMap[MountPathMap.FILEPATHONCONTAINER]}
+                                    name={MountPathMap.FILEPATHONCONTAINER}
+                                />
+                                <Close
+                                    className="icon-dim-24 pointer mt-6 ml-6"
+                                    onClick={() => {
+                                        deleteMountPath(index)
+                                    }}
+                                />
+                            </div>
+                            <div className="pl-200 mb-20">
+                                {formDataErrorObj[activeStageName].steps[selectedTaskIndex]?.inlineStepDetail
+                                    ?.mountPathMap &&
+                                    formDataErrorObj[activeStageName].steps[selectedTaskIndex].inlineStepDetail
+                                        .mountPathMap[index] &&
+                                    !formDataErrorObj[activeStageName].steps[selectedTaskIndex].inlineStepDetail
+                                        .mountPathMap[index].isValid && (
+                                        <span className="flexbox cr-5 mb-4 mt-4 fw-5 fs-11 flexbox">
+                                            <AlertTriangle className="icon-dim-14 mr-5 ml-5 mt-2" />
+                                            <span>
+                                                {
+                                                    formDataErrorObj[activeStageName].steps[selectedTaskIndex]
+                                                        .inlineStepDetail.mountPathMap[index].message
```

Comment:
try to move `formDataErrorObj[activeStageName].steps[selectedTaskIndex].inlineStepDetail.mountPathMap[index]` to some variable above